spk/syncthing/src/dsm-control.sh

```diff
@@ -9,12 +9,17 @@ INSTALL_DIR="/usr/local/${PACKAGE}"
 PATH="${INSTALL_DIR}/bin:${PATH}"
 USER="syncthing"
 SYNCTHING="${INSTALL_DIR}/bin/syncthing"
-CONFIG_DIR="${INSTALL_DIR}/var/"
+CONFIG_DIR="${INSTALL_DIR}/var"
+SYNCTHING_OPTIONS="-home=${CONFIG_DIR}"
 
+# Read additional startup options from /usr/local/syncthing/var/options.conf
```

Comment:
Let's keep inotify stuff in the inotify package. This seems unnecessary (as do most changes in this file).